src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -34,58 +33,75 @@ import {
   EuiComboBox,
   EuiSpacer,
   EuiFormRow,
-  EuiCode,
-  EuiTextArea,
+  EuiFieldNumber,
 } from '@elastic/eui';
 import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { MODEL_TYPES } from '../../../common/model_options';
+
+const DEFAULTS = {
+  model_type: MODEL_TYPES.UNWEIGHTED,
+  alpha: 0.3,
+  beta: 0.1,
+  gamma: 0.1,
+  period: 5,
+  multiplicative: true,
+  window: 10,
```

Comment:
in previous version it was   window: ''. Not sure about 10

Reply:
Changed